app/components/visualizer/sidePanel.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Box } from '../form/box';
+import { Button } from '../form/button';
+
+const Description = styled.div`
+    align-self: normal;
+    display: flex;
+    flex-direction: column;
+    grid-area: lower;
+    padding: 5px;
+    text-align: left;
+
+    & p {
+        text-indent: 2em;
+    }
+`;
+const Title = styled.p`
+    font-size: xx-large;
+    margin: 15px;
+    user-select: none;
+`;
+const UpperContent = styled.div`
+    grid-area: upper;
+`;
+
+export default class SidePanel extends React.Component {
```

Comment:
I am not sure we need to create a class for this. It would be much faster/cleaner to just use a functional component.